benchmark/definition/config.py

```diff
@@ -0,0 +1,29 @@
+"""This module contains the configuration of a benchmark scenario."""
+
+from attrs import define, field
+from attrs.validators import deep_mapping, instance_of
+
+from baybe.recommenders.base import RecommenderProtocol
+from baybe.serialization.mixin import SerialMixin
+
+
+@define(frozen=True)
+class BenchmarkScenarioSettings(SerialMixin):
+    """The configuration for the benchmark settings."""
+
+    batch_size: int = field(validator=instance_of(int))
+    """The batch size for the benchmark."""
+
+    n_doe_iterations: int = field(validator=instance_of(int))
+    """The number of Design of Experiment iterations."""
+
+    n_mc_iterations: int = field(validator=instance_of(int))
+    """The number of Monte Carlo iterations."""
+
+    recommender: dict[str, RecommenderProtocol] = field(
```

Comment:
Why is the recommender part of the `Settings`? This feels a bit arbitrary to me (that is: Why is the `Recommender` part of a setting, but not the search space?). Also, this implies that every benchmark needs to specify a `Recommender` - which is probably a reasonable assumption, but still puzzles me. Let's discuss this in our weekly.

Reply:
I've proposed a design via generics that I can explain in the dev meeting